test/controllers/users_controller_test.rb

```diff
@@ -96,4 +96,21 @@ class UsersControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test "test process multiple scans" do
+    post :process_scan, id: users(:vtk), scan: { scan_items_attributes: {
+      "0": { reservation: reservations(:vtk_tent), pick_up: 1 }
```

Comment:
The test says "multiple scans", but still only 1 item is scanned (although it uses the multi-scan logic).